src/Xamarin.Android.Build.Tasks/Tasks/Aapt.cs

```diff
@@ -102,7 +102,7 @@ bool ManifestIsUpToDate (string manifestFile)
 				File.GetLastWriteTime (AndroidComponentResgenFlagFile) > File.GetLastWriteTime (manifestFile);
 		}
 
-		bool RunAapt (string commandLine)
+		bool RunAapt (string commandLine, IList<Tuple<string, bool>> output)
```

Comment:
Philosophically... `Tuple<string, bool>`?! I wonder if we can use `ValueTuple`, and at least have `(string message, bool isError)`?

Otherwise, *really* not a fan of tuples.

If we can't use `(string message, bool isError)`, perhaps a new struct would be better?